docs/api/prompt/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _api_prompt:
+
+=================
+Prompt API
+=================
+Functions in ``FORMATTER_DICT`` for prompt.
+
+.. toctree::
+    :maxdepth: 1
+
+    gitstatus
+    vc_branch
```

Comment:
No, but the should have module level docstrings and they may have public members in the future.